lib/console_client/console_client.rb

```diff
@@ -46,11 +46,13 @@ def play_game
     def configure_player(player, token)
       @io.clear_screen
       @io.display(@text_provider.get_text(:configure_player, {player: player}))
-      case @io.get_input(['1', '2'], @text_provider.get_text(:invalid_selection))
+      case @io.get_input(['1', '2', '3'], @text_provider.get_text(:invalid_selection))
       when '1'
         Game::Player.create(:human, token)
       when '2'
         Game::Player.create(:easy, token)
+      when '3'
+        Game::Player.create(:easy, token)
```

Comment:
No, no they are not. And I wish I'd seen this comment earlier instead of spending the past couple of hours trying to figure out why my minimax wasn't working in actual game play! Life saver, thanks.